<!DOCTYPE html>
<html prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article#
" lang="en">
<head>
<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1">
<title>Posts about reproducibility infrastructure | Reproducible Science</title>
<link href="../../assets/css/all-nocdn.css" rel="stylesheet" type="text/css">
<meta name="theme-color" content="#5670d4">
<meta name="generator" content="Nikola (getnikola.com)">
<link rel="canonical" href="https://reproduciblescience.org/categories/reproducibility-infrastructure/">
<link rel="favicon" href="favicon.ico" sizes="16x16">
<link rel="next" href="index-1.html" type="text/html">
<!--[if lt IE 9]><script src="../../assets/js/html5.js"></script><![endif]--><link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.1.0/css/solid.css" integrity="sha384-TbilV5Lbhlwdyc4RuIV/JhD8NR+BfMrvz4BL5QFa2we1hQu6wvREr3v6XSRfCTRp" crossorigin="anonymous">
<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.1.0/css/regular.css" integrity="sha384-avJt9MoJH2rB4PKRsJRHZv7yiFZn8LrnXuzvmZoD3fh1aL6aM6s0BBcnCvBe6XSD" crossorigin="anonymous">
<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.1.0/css/brands.css" integrity="sha384-7xAnn7Zm3QC1jFjVc1A6v/toepoG3JXboQYzbM0jrPzou9OFXm/fY6Z/XiIebl/k" crossorigin="anonymous">
<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.1.0/css/fontawesome.css" integrity="sha384-ozJwkrqb90Oa3ZNb+yKFW2lToAWYdTiF1vt8JiH5ptTGHTGcN7qdoR1F95e0kYyG" crossorigin="anonymous">
</head>
<body>
<a href="#content" class="sr-only sr-only-focusable">Skip to main content</a>

<!-- Menubar -->

<nav class="navbar navbar-expand-lg navbar-dark bg-primary static-top mb-4"><div class="container">
<!-- This keeps the margins nice -->
        <a class="navbar-brand" href="https://reproduciblescience.org/">
            <img src="../../logo.png" alt="Reproducible Science" id="logo" class="d-inline-block align-top"><span id="blog-title">Reproducible Science</span>
        </a>
        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#bs-navbar" aria-controls="bs-navbar" aria-expanded="false" aria-label="Toggle navigation">
            <span class="navbar-toggler-icon"></span>
        </button>

        <div class="collapse navbar-collapse" id="bs-navbar" aria-expanded="false">
            <ul class="navbar-nav mr-auto">
<li class="nav-item">
<a href="../../who-we-are/" class="nav-link">Who We Are</a>
         </li>
<li class="nav-item">
<a href="../../contribute/" class="nav-link">Contribute</a>
         </li>
<li class="nav-item">
<a href="../../directory/" class="nav-link">Reproducible Resource Directory</a>

                
            </li>
</ul>
<ul class="navbar-nav navbar-right">
<li class="nav-item">
<a href="https://github.com/vida-nyu/reproducible-science" class="nav-link"><i class="fab fa-github"></i></a>
         </li>
<li class="nav-item">
<a href="https://twitter.com/ReproFeed" class="nav-link"><i class="fab fa-twitter"></i></a>
         </li>
<li class="nav-item">
<a href="https://vida-nyu.github.io/reproducibility-news/feed.rss" class="nav-link"><i class="fas fa-rss"></i></a>

                
                
                
            </li>
</ul>
</div>
<!-- /.navbar-collapse -->
    </div>
<!-- /.container -->
</nav><!-- End of Menubar --><div class="container" id="content" role="main">
    <div class="body-content">
        <!--Body content-->
        
        


    <header><h1>Posts about reproducibility infrastructure</h1>
        <div class="metadata">
            

            

        </div>
    </header><article class="h-entry post-text" itemscope="itemscope" itemtype="http://schema.org/Article"><header><h1 class="p-name entry-title"><a href="https://arxiv.org/abs/1809.07693v1" class="u-url">A Serverless Tool for Platform Agnostic Computational Experiment Management</a></h1>
        <div class="metadata">
            <p class="dateline"><time class="published dt-published" datetime="2018-09-25T00:00:00-04:00" title="25-09-2018">25-09-2018</time></p>  
						<p><a href="mailto:?subject=I%20saw%20this%20and%20thought%20of%20you!&amp;body=A%20Serverless%20Tool%20for%20Platform%20Agnostic%20Computational%20Experiment%20Management;%20https://arxiv.org/abs/1809.07693v1" onclick="ga('send', 'social', 'email', 'share', https://arxiv.org/abs/1809.07693v1); return true;">Email this article</a></p>
						<p><a href="https://twitter.com/share?url=/directory/directory/A-Serverless-Tool-for-Platform-Agnostic-Computational-Experi/);text=A%20Serverless%20Tool%20for%20Platform%20Agnostic%20Computational%20Experiment%20Management,%20https://arxiv.org/abs/1809.07693v1" onclick="ga('send', 'social', 'Twitter', 'tweet', /directory/directory/A-Serverless-Tool-for-Platform-Agnostic-Computational-Experi/); return true;" target="_blank">Tweet this article</a></p> 
        </div>
    </header><div class="e-content entry-content">
    <p>Neuroscience has been carried into the domain of big data and high performance computing (HPC) on the backs of initiatives in data collection and an increasingly compute-intensive tools. While managing HPC experiments requires considerable technical acumen, platforms and standards have been developed to ease this burden on scientists. While web-portals make resources widely accessible, data organizations such as the Brain Imaging Data Structure and tool description languages such as Boutiques provide researchers with a foothold to tackle these problems using their own datasets, pipelines, and environments. While these standards lower the barrier to adoption of HPC and cloud systems for neuroscience applications, they still require the consolidation of disparate domain-specific knowledge. We present Clowdr, a lightweight tool to launch experiments on HPC systems and clouds, record rich execution records, and enable the accessible sharing of experimental summaries and results. Clowdr uniquely sits between web platforms and bare-metal applications for experiment management by preserving the flexibility of do-it-yourself solutions while providing a low barrier for developing, deploying and disseminating neuroscientific analysis.</p>
    </div>
    </article><article class="h-entry post-text" itemscope="itemscope" itemtype="http://schema.org/Article"><header><h1 class="p-name entry-title"><a href="https://link.springer.com/chapter/10.1007/978-3-319-98379-0_3" class="u-url">Classification of Provenance Triples for Scientific Reproducibility: A Comparative Evaluation of Deep Learning Models in the ProvCaRe Project</a></h1>
        <div class="metadata">
            <p class="dateline"><time class="published dt-published" datetime="2018-09-11T00:00:00-04:00" title="11-09-2018">11-09-2018</time></p>  
						<p><a href="mailto:?subject=I%20saw%20this%20and%20thought%20of%20you!&amp;body=Classification%20of%20Provenance%20Triples%20for%20Scientific%20Reproducibility:%20A%20Comparative%20Evaluation%20of%20Deep%20Learning%20Models%20in%20the%20ProvCaRe%20Project;%20https://link.springer.com/chapter/10.1007/978-3-319-98379-0_3" onclick="ga('send', 'social', 'email', 'share', https://link.springer.com/chapter/10.1007/978-3-319-98379-0_3); return true;">Email this article</a></p>
						<p><a href="https://twitter.com/share?url=/directory/directory/Classification-of-Provenance-Triples-for-Scientific-Reproduc/);text=Classification%20of%20Provenance%20Triples%20for%20Scientific%20Reproducibility:%20A%20Comparative%20Evaluation%20of%20Deep%20Learning%20Models%20in%20the%20ProvCaRe%20Project,%20https://link.springer.com/chapter/10.1007/978-3-319-98379-0_3" onclick="ga('send', 'social', 'Twitter', 'tweet', /directory/directory/Classification-of-Provenance-Triples-for-Scientific-Reproduc/); return true;" target="_blank">Tweet this article</a></p> 
        </div>
    </header><div class="e-content entry-content">
    <p>Scientific reproducibility is key to the advancement of science as researchers can build on sound and validated results to design new research studies. However, recent studies in biomedical research have highlighted key challenges in scientific reproducibility as more than 70% of researchers in a survey of more than 1500 participants were not able to reproduce results from other groups and 50% of researchers were not able to reproduce their own experiments. Provenance metadata is a key component of scientific reproducibility and as part of the Provenance for Clinical and Health Research (ProvCaRe) project, we have: (1) identified and modeled important provenance terms associated with a biomedical research study in the S3 model (formalized in the ProvCaRe ontology); (2) developed a new natural language processing (NLP) workflow to identify and extract provenance metadata from published articles describing biomedical research studies; and (3) developed the ProvCaRe knowledge repository to enable users to query and explore provenance of research studies using the S3 model. However, a key challenge in this project is the automated classification of provenance metadata extracted by the NLP workflow according to the S3 model and its subsequent querying in the ProvCaRe knowledge repository. In this paper, we describe the development and comparative evaluation of deep learning techniques for multi-class classification of structured provenance metadata extracted from biomedical literature using 12 different categories of provenance terms represented in the S3 model. We describe the application of the Long Term Short Memory (LSTM) network, which has the highest classification accuracy of 86% in our evaluation, to classify more than 48 million provenance triples in the ProvCaRe knowledge repository (available at: https://provcare.case.edu/).</p>
    </div>
    </article><article class="h-entry post-text" itemscope="itemscope" itemtype="http://schema.org/Article"><header><h1 class="p-name entry-title"><a href="https://dl.acm.org/citation.cfm?id=3275411" class="u-url">Reproducibility study of a PDEVS model application to fire spreading</a></h1>
        <div class="metadata">
            <p class="dateline"><time class="published dt-published" datetime="2018-09-11T00:00:00-04:00" title="11-09-2018">11-09-2018</time></p>  
						<p><a href="mailto:?subject=I%20saw%20this%20and%20thought%20of%20you!&amp;body=Reproducibility%20study%20of%20a%20PDEVS%20model%20application%20to%20fire%20spreading;%20https://dl.acm.org/citation.cfm?id=3275411" onclick="ga('send', 'social', 'email', 'share', https://dl.acm.org/citation.cfm?id=3275411); return true;">Email this article</a></p>
						<p><a href="https://twitter.com/share?url=/directory/directory/Reproducibility-study-of-a-PDEVS-model-application-to-fire-s/);text=Reproducibility%20study%20of%20a%20PDEVS%20model%20application%20to%20fire%20spreading,%20https://dl.acm.org/citation.cfm?id=3275411" onclick="ga('send', 'social', 'Twitter', 'tweet', /directory/directory/Reproducibility-study-of-a-PDEVS-model-application-to-fire-s/); return true;" target="_blank">Tweet this article</a></p> 
        </div>
    </header><div class="e-content entry-content">
    <p>The results of a scientific experiment have to be reproduced to be valid. The scientific method is well known in experimental sciences but it is not always the case for computer scientists. Recent publications and studies has shown that there is a significant reproducibility crisis in Biology and Medicine. This problem has also been demonstrated for hundreds of publications in computer science where only a limited set of publication results could be reproduced. In this paper we present the reproducibility challenge and we examine the reproducibility of a Parallel Discrete Event System Specification (PDEVS) model with two different execution frameworks.</p>
    </div>
    </article><article class="h-entry post-text" itemscope="itemscope" itemtype="http://schema.org/Article"><header><h1 class="p-name entry-title"><a href="https://gatesopenresearch.org/articles/2-31/v1" class="u-url">DataPackageR: Reproducible data preprocessing, standardization and sharing using R/Bioconductor for collaborative data analysis</a></h1>
        <div class="metadata">
            <p class="dateline"><time class="published dt-published" datetime="2018-07-05T00:00:00-04:00" title="05-07-2018">05-07-2018</time></p>  
						<p><a href="mailto:?subject=I%20saw%20this%20and%20thought%20of%20you!&amp;body=DataPackageR:%20Reproducible%20data%20preprocessing,%20standardization%20and%20sharing%20using%20R/Bioconductor%20for%20collaborative%20data%20analysis;%20https://gatesopenresearch.org/articles/2-31/v1" onclick="ga('send', 'social', 'email', 'share', https://gatesopenresearch.org/articles/2-31/v1); return true;">Email this article</a></p>
						<p><a href="https://twitter.com/share?url=/directory/directory/DataPackageR-Reproducible-data-preprocessing-standardization/);text=DataPackageR:%20Reproducible%20data%20preprocessing,%20standardization%20and%20sharing%20using%20R/Bioconductor%20for%20collaborative%20data%20analysis,%20https://gatesopenresearch.org/articles/2-31/v1" onclick="ga('send', 'social', 'Twitter', 'tweet', /directory/directory/DataPackageR-Reproducible-data-preprocessing-standardization/); return true;" target="_blank">Tweet this article</a></p> 
        </div>
    </header><div class="e-content entry-content">
    <p>A central tenet of reproducible research is that scientific results are published along with the underlying data and software code necessary to reproduce and verify the findings. A host of tools and software have been released that facilitate such work-flows and scientific journals have increasingly demanded that code and primary data be made available with publications. There has been little practical advice on implementing reproducible research work-flows for large ’omics’ or systems biology data sets used by teams of analysts working in collaboration. In such instances it is important to ensure all analysts use the same version of a data set for their analyses. Yet, instantiating relational databases and standard operating procedures can be unwieldy, with high "startup" costs and poor adherence to procedures when they deviate substantially from an analyst’s usual work-flow. Ideally a reproducible research work-flow should fit naturally into an individual’s existing work-flow, with minimal disruption. Here, we provide an overview of how we have leveraged popular open source tools, including Bioconductor, Rmarkdown, git version control, R, and specifically R’s package system combined with a new tool DataPackageR, to implement a lightweight reproducible research work-flow for preprocessing large data sets, suitable for sharing among small-to-medium sized teams of computational scientists. Our primary contribution is the DataPackageR tool, which decouples time-consuming data processing from data analysis while leaving a traceable record of how raw data is processed into analysis-ready data sets. The software ensures packaged data objects are properly documented and performs checksum verification of these along with basic package version management, and importantly, leaves a record of data processing code in the form of package vignettes. Our group has implemented this work-flow to manage, analyze and report on pre-clinical immunological trial data from multi-center, multi-assay studies for the past three years.</p>
    </div>
    </article><article class="h-entry post-text" itemscope="itemscope" itemtype="http://schema.org/Article"><header><h1 class="p-name entry-title"><a href="https://arxiv.org/pdf/1806.09525.pdf" class="u-url">How to Read a Research Compendium</a></h1>
        <div class="metadata">
            <p class="dateline"><time class="published dt-published" datetime="2018-06-30T00:00:00-04:00" title="30-06-2018">30-06-2018</time></p>  
						<p><a href="mailto:?subject=I%20saw%20this%20and%20thought%20of%20you!&amp;body=How%20to%20Read%20a%20Research%20Compendium;%20https://arxiv.org/pdf/1806.09525.pdf" onclick="ga('send', 'social', 'email', 'share', https://arxiv.org/pdf/1806.09525.pdf); return true;">Email this article</a></p>
						<p><a href="https://twitter.com/share?url=/directory/directory/How-to-Read-a-Research-Compendium/);text=How%20to%20Read%20a%20Research%20Compendium,%20https://arxiv.org/pdf/1806.09525.pdf" onclick="ga('send', 'social', 'Twitter', 'tweet', /directory/directory/How-to-Read-a-Research-Compendium/); return true;" target="_blank">Tweet this article</a></p> 
        </div>
    </header><div class="e-content entry-content">
    <p>Researchers spend a great deal of time reading research papers. Keshav (2012) provides a three-pass method to researchers to improve their reading skills. This article extends Keshav's method for reading a research compendium. Research compendia are an increasingly used form of publication, which packages not only the research paper's text and figures, but also all data and software for better reproducibility. We introduce the existing conventions for research compendia and suggest how to utilise their shared properties in a structured reading process. Unlike the original, this article is not build upon a long history but intends to provide guidance at the outset of an emerging practice.</p>
    </div>
    </article><article class="h-entry post-text" itemscope="itemscope" itemtype="http://schema.org/Article"><header><h1 class="p-name entry-title"><a href="https://watermark.silverchair.com/giy072.pdf?token=AQECAHi208BE49Ooan9kkhW_Ercy7Dm3ZL_9Cf3qfKAc485ysgAAAdIwggHOBgkqhkiG9w0BBwagggG_MIIBuwIBADCCAbQGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQM9e5qZs5DsbcT7x6rAgEQgIIBhSLDG-1fl8D5GYdRDo-2wBE35B-JUWT1SELphgsGS_7CBSCg9GQc7B81NfOMex-RSvfwHbIBIbz5nksrXpTSpkpqjAMuzfrTNxDmLykrRHKT77Aor9BJNfACV3E2eOy1GXT6B08kbo77o85nn8G8vdSE9Qf7Dv-ACvv2bEi07bZrQ2WPC14oEFIOWKmorKXrhIcQrI7CrU3MyoypLWGhEsLh4BnTgSbs13V4yTpI7FbPur6wUMVBP81cru_Ud33rwrH4GKKADD8RYRMMwuN3RU_ZSl-XVDe2ph6RBw6KPfnA_imkXp8SRjPfsy5xieC8JJo4RqnQKsrMp895HOc3OI5nD0gyPJqWpPqqBOdDGb5hPaFpBfT_bJidP6xsIrQP9zleZAHSsEewOPfDOzCnycVPoJIhcyWCpTu3McLAu3IRny3XZKa2EtxvgRR9Tgcm7s-WHYh1MxvlabnKaboJE08SiPCr_T6CCKEkUhTovpwja1NrmhuQM5HYPY9ZRjrPd4-Nt2xG" class="u-url">YAMP: a containerised workflow enabling reproducibility in metagenomics research</a></h1>
        <div class="metadata">
            <p class="dateline"><time class="published dt-published" datetime="2018-06-19T00:00:00-04:00" title="19-06-2018">19-06-2018</time></p>  
						<p><a href="mailto:?subject=I%20saw%20this%20and%20thought%20of%20you!&amp;body=YAMP:%20a%20containerised%20workflow%20enabling%20reproducibility%20in%20metagenomics%20research;%20https://watermark.silverchair.com/giy072.pdf?token=AQECAHi208BE49Ooan9kkhW_Ercy7Dm3ZL_9Cf3qfKAc485ysgAAAdIwggHOBgkqhkiG9w0BBwagggG_MIIBuwIBADCCAbQGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQM9e5qZs5DsbcT7x6rAgEQgIIBhSLDG-1fl8D5GYdRDo-2wBE35B-JUWT1SELphgsGS_7CBSCg9GQc7B81NfOMex-RSvfwHbIBIbz5nksrXpTSpkpqjAMuzfrTNxDmLykrRHKT77Aor9BJNfACV3E2eOy1GXT6B08kbo77o85nn8G8vdSE9Qf7Dv-ACvv2bEi07bZrQ2WPC14oEFIOWKmorKXrhIcQrI7CrU3MyoypLWGhEsLh4BnTgSbs13V4yTpI7FbPur6wUMVBP81cru_Ud33rwrH4GKKADD8RYRMMwuN3RU_ZSl-XVDe2ph6RBw6KPfnA_imkXp8SRjPfsy5xieC8JJo4RqnQKsrMp895HOc3OI5nD0gyPJqWpPqqBOdDGb5hPaFpBfT_bJidP6xsIrQP9zleZAHSsEewOPfDOzCnycVPoJIhcyWCpTu3McLAu3IRny3XZKa2EtxvgRR9Tgcm7s-WHYh1MxvlabnKaboJE08SiPCr_T6CCKEkUhTovpwja1NrmhuQM5HYPY9ZRjrPd4-Nt2xG" onclick="ga('send', 'social', 'email', 'share', https://watermark.silverchair.com/giy072.pdf?token=AQECAHi208BE49Ooan9kkhW_Ercy7Dm3ZL_9Cf3qfKAc485ysgAAAdIwggHOBgkqhkiG9w0BBwagggG_MIIBuwIBADCCAbQGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQM9e5qZs5DsbcT7x6rAgEQgIIBhSLDG-1fl8D5GYdRDo-2wBE35B-JUWT1SELphgsGS_7CBSCg9GQc7B81NfOMex-RSvfwHbIBIbz5nksrXpTSpkpqjAMuzfrTNxDmLykrRHKT77Aor9BJNfACV3E2eOy1GXT6B08kbo77o85nn8G8vdSE9Qf7Dv-ACvv2bEi07bZrQ2WPC14oEFIOWKmorKXrhIcQrI7CrU3MyoypLWGhEsLh4BnTgSbs13V4yTpI7FbPur6wUMVBP81cru_Ud33rwrH4GKKADD8RYRMMwuN3RU_ZSl-XVDe2ph6RBw6KPfnA_imkXp8SRjPfsy5xieC8JJo4RqnQKsrMp895HOc3OI5nD0gyPJqWpPqqBOdDGb5hPaFpBfT_bJidP6xsIrQP9zleZAHSsEewOPfDOzCnycVPoJIhcyWCpTu3McLAu3IRny3XZKa2EtxvgRR9Tgcm7s-WHYh1MxvlabnKaboJE08SiPCr_T6CCKEkUhTovpwja1NrmhuQM5HYPY9ZRjrPd4-Nt2xG); return true;">Email this article</a></p>
						<p><a href="https://twitter.com/share?url=/directory/directory/YAMP-a-containerised-workflow-enabling-reproducibility-in-me/);text=YAMP:%20a%20containerised%20workflow%20enabling%20reproducibility%20in%20metagenomics%20research,%20https://watermark.silverchair.com/giy072.pdf?token=AQECAHi208BE49Ooan9kkhW_Ercy7Dm3ZL_9Cf3qfKAc485ysgAAAdIwggHOBgkqhkiG9w0BBwagggG_MIIBuwIBADCCAbQGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQM9e5qZs5DsbcT7x6rAgEQgIIBhSLDG-1fl8D5GYdRDo-2wBE35B-JUWT1SELphgsGS_7CBSCg9GQc7B81NfOMex-RSvfwHbIBIbz5nksrXpTSpkpqjAMuzfrTNxDmLykrRHKT77Aor9BJNfACV3E2eOy1GXT6B08kbo77o85nn8G8vdSE9Qf7Dv-ACvv2bEi07bZrQ2WPC14oEFIOWKmorKXrhIcQrI7CrU3MyoypLWGhEsLh4BnTgSbs13V4yTpI7FbPur6wUMVBP81cru_Ud33rwrH4GKKADD8RYRMMwuN3RU_ZSl-XVDe2ph6RBw6KPfnA_imkXp8SRjPfsy5xieC8JJo4RqnQKsrMp895HOc3OI5nD0gyPJqWpPqqBOdDGb5hPaFpBfT_bJidP6xsIrQP9zleZAHSsEewOPfDOzCnycVPoJIhcyWCpTu3McLAu3IRny3XZKa2EtxvgRR9Tgcm7s-WHYh1MxvlabnKaboJE08SiPCr_T6CCKEkUhTovpwja1NrmhuQM5HYPY9ZRjrPd4-Nt2xG" onclick="ga('send', 'social', 'Twitter', 'tweet', /directory/directory/YAMP-a-containerised-workflow-enabling-reproducibility-in-me/); return true;" target="_blank">Tweet this article</a></p> 
        </div>
    </header><div class="e-content entry-content">
    <p>YAMP is a user-friendly workflow that enables the analysis of whole shotgun metagenomic data while using containerisation to ensure computational reproducibility and facilitate collaborative research. YAMP can be executed on any UNIX-like system, and offers seamless support for multiple job schedulers as well as for Amazon AWS cloud. Although YAMP has been developed to be ready-to-use by non-experts, bioinformaticians will appreciate its flexibility, modularisation, and simple customisation. The YAMP script, parameters, and documentation are available at https://github.com/alesssia/YAMP.</p>
    </div>
    </article><nav aria-label="Page navigation"><ul class="pagination pagination-lg justify-content-center">
<li class="page-item disabled"><a href="#" class="page-link" aria-label="Newer posts"><span aria-hidden="true">«</span></a></li>
      <li class="page-item active"><a href="." class="page-link">1 <span class="sr-only">(current)</span></a></li>
      <li class="page-item "><a href="index-1.html" class="page-link">2</a></li>
      <li class="page-item "><a href="index-2.html" class="page-link">3</a></li>
      <li class="page-item "><a href="index-3.html" class="page-link">4</a></li>
      <li class="page-item "><a href="index-4.html" class="page-link">5</a></li>
      <li class="page-item "><a href="index-5.html" class="page-link">6</a></li>
      <li class="page-item "><a href="index-6.html" class="page-link">7</a></li>
      <li class="page-item"><a href="index-1.html" class="page-link" aria-label="Older posts"><span aria-hidden="true">»</span></a></li>
  </ul></nav><!--End of body content--><footer id="footer" class="mt-4 mb-4">
            2019 <a href="mailto:vicky.steeves@nyu.edu">NYU Reproducibility WG</a> | <a href="https://github.com/vida-nyu/reproducible-science"><i class="fab fa-github"></i></a> <a href="https://twitter.com/ReproFeed"><i class="fab fa-twitter"></i></a> <a href="https://vida-nyu.github.io/reproducibility-news/feed.rss"><i class="fas fa-rss"></i></a>
            
        </footer>
</div>
</div>


        <script src="../../assets/js/all-nocdn.js"></script><!-- fancy dates --><script>
    moment.locale("en");
    fancydates(0, "DD:MM:YYYY");
    </script><!-- end fancy dates --><script>
    baguetteBox.run('div#content', {
        ignoreClass: 'islink',
        captions: function(element) {
            return element.getElementsByTagName('img')[0].alt;
    }});
    </script><script>(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
  ga('create', 'UA-86255717-1', 'auto');
ga('send', 'pageview');</script>
</body>
</html>
