<!DOCTYPE html>
<html prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article#
" lang="en">
<head>
<meta charset="utf-8">
<meta name="description" content="Promoting reproducibility and openness!">
<meta name="viewport" content="width=device-width, initial-scale=1">
<title>Reproducible Science (old posts, page 16) | Reproducible Science</title>
<link href="../assets/css/all-nocdn.css" rel="stylesheet" type="text/css">
<meta name="theme-color" content="#5670d4">
<meta name="generator" content="Nikola (getnikola.com)">
<link rel="canonical" href="https://reproduciblescience.org/directory/index-16.html">
<link rel="favicon" href="favicon.ico" sizes="16x16">
<link rel="prev" href="index-15.html" type="text/html">
<link rel="next" href="index-17.html" type="text/html">
<!--[if lt IE 9]><script src="../assets/js/html5.js"></script><![endif]--><link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.1.0/css/solid.css" integrity="sha384-TbilV5Lbhlwdyc4RuIV/JhD8NR+BfMrvz4BL5QFa2we1hQu6wvREr3v6XSRfCTRp" crossorigin="anonymous">
<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.1.0/css/regular.css" integrity="sha384-avJt9MoJH2rB4PKRsJRHZv7yiFZn8LrnXuzvmZoD3fh1aL6aM6s0BBcnCvBe6XSD" crossorigin="anonymous">
<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.1.0/css/brands.css" integrity="sha384-7xAnn7Zm3QC1jFjVc1A6v/toepoG3JXboQYzbM0jrPzou9OFXm/fY6Z/XiIebl/k" crossorigin="anonymous">
<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.1.0/css/fontawesome.css" integrity="sha384-ozJwkrqb90Oa3ZNb+yKFW2lToAWYdTiF1vt8JiH5ptTGHTGcN7qdoR1F95e0kYyG" crossorigin="anonymous">
</head>
<body>
<a href="#content" class="sr-only sr-only-focusable">Skip to main content</a>

<!-- Menubar -->

<nav class="navbar navbar-expand-lg navbar-dark bg-primary static-top mb-4"><div class="container">
<!-- This keeps the margins nice -->
        <a class="navbar-brand" href="https://reproduciblescience.org/">
            <img src="../logo.png" alt="Reproducible Science" id="logo" class="d-inline-block align-top"><span id="blog-title">Reproducible Science</span>
        </a>
        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#bs-navbar" aria-controls="bs-navbar" aria-expanded="false" aria-label="Toggle navigation">
            <span class="navbar-toggler-icon"></span>
        </button>

        <div class="collapse navbar-collapse" id="bs-navbar" aria-expanded="false">
            <ul class="navbar-nav mr-auto">
<li class="nav-item">
<a href="../who-we-are/" class="nav-link">Who We Are</a>
         </li>
<li class="nav-item">
<a href="../contribute/" class="nav-link">Contribute</a>
         </li>
<li class="nav-item">
<a href="." class="nav-link">Reproducible Resource Directory</a>

                
            </li>
</ul>
<ul class="navbar-nav navbar-right">
<li class="nav-item">
<a href="https://github.com/vida-nyu/reproducible-science" class="nav-link"><i class="fab fa-github"></i></a>
         </li>
<li class="nav-item">
<a href="https://twitter.com/ReproFeed" class="nav-link"><i class="fab fa-twitter"></i></a>
         </li>
<li class="nav-item">
<a href="https://vida-nyu.github.io/reproducibility-news/feed.rss" class="nav-link"><i class="fas fa-rss"></i></a>

                
                
                
            </li>
</ul>
</div>
<!-- /.navbar-collapse -->
    </div>
<!-- /.container -->
</nav><!-- End of Menubar --><div class="container" id="content" role="main">
    <div class="body-content">
        <!--Body content-->
        
        

    


    

    <article class="h-entry post-text" itemscope="itemscope" itemtype="http://schema.org/Article"><header><h1 class="p-name entry-title"><a href="https://arxiv.org/pdf/1806.06145.pdf" class="u-url">Teaching computational reproducibility for neuroimaging</a></h1>
        <div class="metadata">
            <p class="dateline"><time class="published dt-published" datetime="2018-06-23T00:00:00-04:00" title="23-06-2018">23-06-2018</time></p>  
						<p><a href="mailto:?subject=I%20saw%20this%20and%20thought%20of%20you!&amp;body=Teaching%20computational%20reproducibility%20for%20neuroimaging;%20https://arxiv.org/pdf/1806.06145.pdf" onclick="ga('send', 'social', 'email', 'share', https://arxiv.org/pdf/1806.06145.pdf); return true;">Email this article</a></p>
						<p><a href="https://twitter.com/share?url=/directory/directory/Teaching-computational-reproducibility-for-neuroimaging/);text=Teaching%20computational%20reproducibility%20for%20neuroimaging,%20https://arxiv.org/pdf/1806.06145.pdf" onclick="ga('send', 'social', 'Twitter', 'tweet', /directory/directory/Teaching-computational-reproducibility-for-neuroimaging/); return true;" target="_blank">Tweet this article</a></p> 
        </div>
    </header><div class="e-content entry-content">
    <p>We describe a project-based introduction to reproducible and collaborative neuroimaging analysis. Traditional teaching on neuroimaging usually consists of a series of lectures that emphasize the big picture rather than the foundations on which the techniques are based. The lectures are often paired with practical workshops in which students run imaging analyses using the graphical interface of specific neu-roimaging software packages. Our experience suggests that this combination leaves the student with asuperficial understanding of the underlying ideas, and an informal, inefficient, and inaccurate approach to  analysis.   To  address  these  problems,  we  based  our  course  around a substantial  open-ended  group project.  This allowed us to teach:  (a) computational tools to ensure computationally reproducible work,such as the Unix command line, structured code, version control, automated testing, and code reviewand (b) a clear understanding of the statistical techniques used for a basic analysis of a single run in an MRI scanner.  The emphasis we put on the group project showed the importance of standard computational tools for accuracy, efficiency, and collaboration.  The projects were broadly successful in engagingstudents in working reproducibly on real scientific questions.  We propose that a course on this modelshould be the foundation for future programs in neuroimaging.  We believe it will also serve as a modelfor teaching efficient and reproducible research in other fields of computational science</p>
		<div id="index-tags">
		  <ul class="tags">
			Tagged:
		  	<li class="tag"><a href="../categories/reproducible-paper/">reproducible paper</a></li>          
		  </ul>
</div>
    </div>
    </article><article class="h-entry post-text" itemscope="itemscope" itemtype="http://schema.org/Article"><header><h1 class="p-name entry-title"><a href="https://openreview.net/pdf?id=Syg4NHz4eQ" class="u-url">The RAMP framework: from reproducibility to transparency in the design and optimization of scientific workflows</a></h1>
        <div class="metadata">
            <p class="dateline"><time class="published dt-published" datetime="2018-06-19T00:00:00-04:00" title="19-06-2018">19-06-2018</time></p>  
						<p><a href="mailto:?subject=I%20saw%20this%20and%20thought%20of%20you!&amp;body=The%20RAMP%20framework:%20from%20reproducibility%20to%20transparency%20in%20the%20design%20and%20optimization%20of%20scientific%20workflows;%20https://openreview.net/pdf?id=Syg4NHz4eQ" onclick="ga('send', 'social', 'email', 'share', https://openreview.net/pdf?id=Syg4NHz4eQ); return true;">Email this article</a></p>
						<p><a href="https://twitter.com/share?url=/directory/directory/The-RAMP-framework-from-reproducibility-to-transparency-in-t/);text=The%20RAMP%20framework:%20from%20reproducibility%20to%20transparency%20in%20the%20design%20and%20optimization%20of%20scientific%20workflows,%20https://openreview.net/pdf?id=Syg4NHz4eQ" onclick="ga('send', 'social', 'Twitter', 'tweet', /directory/directory/The-RAMP-framework-from-reproducibility-to-transparency-in-t/); return true;" target="_blank">Tweet this article</a></p> 
        </div>
    </header><div class="e-content entry-content">
    <p>The RAMP (Rapid Analytics and Model Prototyping) is a software and project management tool developed by the Paris-Saclay Center for Data Science. The original goal was to accelerate the adoption of high-quality data science solutions for domain science problems by running rapid collaborative prototyping sessions. Today it is a full-blown data science project management tool promoting reproducibility, fair and transparent model evaluation, and democratization of data science.  We have used the framework for setting up and solving about twenty scientific problems, for organizing scientific sub-communities around these events, and for training novice data scientists.</p>
		<div id="index-tags">
		  <ul class="tags">
			Tagged:
		  	<li class="tag"><a href="../categories/reproducible-paper/">reproducible paper</a></li>          
		  </ul>
</div>
    </div>
    </article><article class="h-entry post-text" itemscope="itemscope" itemtype="http://schema.org/Article"><header><h1 class="p-name entry-title"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5998676/" class="u-url">Three Dimensions of Reproducibility in Natural Language Processing</a></h1>
        <div class="metadata">
            <p class="dateline"><time class="published dt-published" datetime="2018-06-19T00:00:00-04:00" title="19-06-2018">19-06-2018</time></p>  
						<p><a href="mailto:?subject=I%20saw%20this%20and%20thought%20of%20you!&amp;body=Three%20Dimensions%20of%20Reproducibility%20in%20Natural%20Language%20Processing;%20https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5998676/" onclick="ga('send', 'social', 'email', 'share', https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5998676/); return true;">Email this article</a></p>
						<p><a href="https://twitter.com/share?url=/directory/directory/Three-Dimensions-of-Reproducibility-in-Natural-Language-Proc/);text=Three%20Dimensions%20of%20Reproducibility%20in%20Natural%20Language%20Processing,%20https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5998676/" onclick="ga('send', 'social', 'Twitter', 'tweet', /directory/directory/Three-Dimensions-of-Reproducibility-in-Natural-Language-Proc/); return true;" target="_blank">Tweet this article</a></p> 
        </div>
    </header><div class="e-content entry-content">
    <p>Despite considerable recent attention to problems with reproducibility of scientific research, there is a striking lack of agreement about the definition of the term. That is a problem, because the lack of a consensus definition makes it difficult to compare studies of reproducibility, and thus to have even a broad overview of the state of the issue in natural language processing. This paper proposes an ontology of reproducibility in that field. Its goal is to enhance both future research and communication about the topic, and retrospective meta-analyses. We show that three dimensions of reproducibility, corresponding to three kinds of claims in natural language processing papers, can account for a variety of types of research reports. These dimensions are reproducibility of a conclusion, of a finding, and of a value. Three biomedical natural language processing papers by the authors of this paper are analyzed with respect to these dimensions.</p>
		<div id="index-tags">
		  <ul class="tags">
			Tagged:
		  	<li class="tag"><a href="../categories/reproducible-paper/">reproducible paper</a></li>          
		  </ul>
</div>
    </div>
    </article><article class="h-entry post-text" itemscope="itemscope" itemtype="http://schema.org/Article"><header><h1 class="p-name entry-title"><a href="https://watermark.silverchair.com/giy072.pdf?token=AQECAHi208BE49Ooan9kkhW_Ercy7Dm3ZL_9Cf3qfKAc485ysgAAAdIwggHOBgkqhkiG9w0BBwagggG_MIIBuwIBADCCAbQGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQM9e5qZs5DsbcT7x6rAgEQgIIBhSLDG-1fl8D5GYdRDo-2wBE35B-JUWT1SELphgsGS_7CBSCg9GQc7B81NfOMex-RSvfwHbIBIbz5nksrXpTSpkpqjAMuzfrTNxDmLykrRHKT77Aor9BJNfACV3E2eOy1GXT6B08kbo77o85nn8G8vdSE9Qf7Dv-ACvv2bEi07bZrQ2WPC14oEFIOWKmorKXrhIcQrI7CrU3MyoypLWGhEsLh4BnTgSbs13V4yTpI7FbPur6wUMVBP81cru_Ud33rwrH4GKKADD8RYRMMwuN3RU_ZSl-XVDe2ph6RBw6KPfnA_imkXp8SRjPfsy5xieC8JJo4RqnQKsrMp895HOc3OI5nD0gyPJqWpPqqBOdDGb5hPaFpBfT_bJidP6xsIrQP9zleZAHSsEewOPfDOzCnycVPoJIhcyWCpTu3McLAu3IRny3XZKa2EtxvgRR9Tgcm7s-WHYh1MxvlabnKaboJE08SiPCr_T6CCKEkUhTovpwja1NrmhuQM5HYPY9ZRjrPd4-Nt2xG" class="u-url">YAMP: a containerised workflow enabling reproducibility in metagenomics research</a></h1>
        <div class="metadata">
            <p class="dateline"><time class="published dt-published" datetime="2018-06-19T00:00:00-04:00" title="19-06-2018">19-06-2018</time></p>  
						<p><a href="mailto:?subject=I%20saw%20this%20and%20thought%20of%20you!&amp;body=YAMP:%20a%20containerised%20workflow%20enabling%20reproducibility%20in%20metagenomics%20research;%20https://watermark.silverchair.com/giy072.pdf?token=AQECAHi208BE49Ooan9kkhW_Ercy7Dm3ZL_9Cf3qfKAc485ysgAAAdIwggHOBgkqhkiG9w0BBwagggG_MIIBuwIBADCCAbQGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQM9e5qZs5DsbcT7x6rAgEQgIIBhSLDG-1fl8D5GYdRDo-2wBE35B-JUWT1SELphgsGS_7CBSCg9GQc7B81NfOMex-RSvfwHbIBIbz5nksrXpTSpkpqjAMuzfrTNxDmLykrRHKT77Aor9BJNfACV3E2eOy1GXT6B08kbo77o85nn8G8vdSE9Qf7Dv-ACvv2bEi07bZrQ2WPC14oEFIOWKmorKXrhIcQrI7CrU3MyoypLWGhEsLh4BnTgSbs13V4yTpI7FbPur6wUMVBP81cru_Ud33rwrH4GKKADD8RYRMMwuN3RU_ZSl-XVDe2ph6RBw6KPfnA_imkXp8SRjPfsy5xieC8JJo4RqnQKsrMp895HOc3OI5nD0gyPJqWpPqqBOdDGb5hPaFpBfT_bJidP6xsIrQP9zleZAHSsEewOPfDOzCnycVPoJIhcyWCpTu3McLAu3IRny3XZKa2EtxvgRR9Tgcm7s-WHYh1MxvlabnKaboJE08SiPCr_T6CCKEkUhTovpwja1NrmhuQM5HYPY9ZRjrPd4-Nt2xG" onclick="ga('send', 'social', 'email', 'share', https://watermark.silverchair.com/giy072.pdf?token=AQECAHi208BE49Ooan9kkhW_Ercy7Dm3ZL_9Cf3qfKAc485ysgAAAdIwggHOBgkqhkiG9w0BBwagggG_MIIBuwIBADCCAbQGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQM9e5qZs5DsbcT7x6rAgEQgIIBhSLDG-1fl8D5GYdRDo-2wBE35B-JUWT1SELphgsGS_7CBSCg9GQc7B81NfOMex-RSvfwHbIBIbz5nksrXpTSpkpqjAMuzfrTNxDmLykrRHKT77Aor9BJNfACV3E2eOy1GXT6B08kbo77o85nn8G8vdSE9Qf7Dv-ACvv2bEi07bZrQ2WPC14oEFIOWKmorKXrhIcQrI7CrU3MyoypLWGhEsLh4BnTgSbs13V4yTpI7FbPur6wUMVBP81cru_Ud33rwrH4GKKADD8RYRMMwuN3RU_ZSl-XVDe2ph6RBw6KPfnA_imkXp8SRjPfsy5xieC8JJo4RqnQKsrMp895HOc3OI5nD0gyPJqWpPqqBOdDGb5hPaFpBfT_bJidP6xsIrQP9zleZAHSsEewOPfDOzCnycVPoJIhcyWCpTu3McLAu3IRny3XZKa2EtxvgRR9Tgcm7s-WHYh1MxvlabnKaboJE08SiPCr_T6CCKEkUhTovpwja1NrmhuQM5HYPY9ZRjrPd4-Nt2xG); return true;">Email this article</a></p>
						<p><a href="https://twitter.com/share?url=/directory/directory/YAMP-a-containerised-workflow-enabling-reproducibility-in-me/);text=YAMP:%20a%20containerised%20workflow%20enabling%20reproducibility%20in%20metagenomics%20research,%20https://watermark.silverchair.com/giy072.pdf?token=AQECAHi208BE49Ooan9kkhW_Ercy7Dm3ZL_9Cf3qfKAc485ysgAAAdIwggHOBgkqhkiG9w0BBwagggG_MIIBuwIBADCCAbQGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQM9e5qZs5DsbcT7x6rAgEQgIIBhSLDG-1fl8D5GYdRDo-2wBE35B-JUWT1SELphgsGS_7CBSCg9GQc7B81NfOMex-RSvfwHbIBIbz5nksrXpTSpkpqjAMuzfrTNxDmLykrRHKT77Aor9BJNfACV3E2eOy1GXT6B08kbo77o85nn8G8vdSE9Qf7Dv-ACvv2bEi07bZrQ2WPC14oEFIOWKmorKXrhIcQrI7CrU3MyoypLWGhEsLh4BnTgSbs13V4yTpI7FbPur6wUMVBP81cru_Ud33rwrH4GKKADD8RYRMMwuN3RU_ZSl-XVDe2ph6RBw6KPfnA_imkXp8SRjPfsy5xieC8JJo4RqnQKsrMp895HOc3OI5nD0gyPJqWpPqqBOdDGb5hPaFpBfT_bJidP6xsIrQP9zleZAHSsEewOPfDOzCnycVPoJIhcyWCpTu3McLAu3IRny3XZKa2EtxvgRR9Tgcm7s-WHYh1MxvlabnKaboJE08SiPCr_T6CCKEkUhTovpwja1NrmhuQM5HYPY9ZRjrPd4-Nt2xG" onclick="ga('send', 'social', 'Twitter', 'tweet', /directory/directory/YAMP-a-containerised-workflow-enabling-reproducibility-in-me/); return true;" target="_blank">Tweet this article</a></p> 
        </div>
    </header><div class="e-content entry-content">
    <p>YAMP is a user-friendly workflow that enables the analysis of whole shotgun metagenomic data while using containerisation to ensure computational reproducibility and facilitate collaborative research. YAMP can be executed on any UNIX-like system, and offers seamless support for multiple job schedulers as well as for Amazon AWS cloud. Although YAMP has been developed to be ready-to-use by non-experts, bioinformaticians will appreciate its flexibility, modularisation, and simple customisation. The YAMP script, parameters, and documentation are available at https://github.com/alesssia/YAMP.</p>
		<div id="index-tags">
		  <ul class="tags">
			Tagged:
		  	<li class="tag"><a href="../categories/reproducibility-infrastructure/">reproducibility infrastructure</a></li>          
		  	<li class="tag"><a href="../categories/reproducible-paper/">reproducible paper</a></li>          
		  </ul>
</div>
    </div>
    </article><article class="h-entry post-text" itemscope="itemscope" itemtype="http://schema.org/Article"><header><h1 class="p-name entry-title"><a href="https://dl.acm.org/citation.cfm?doid=3214239.3214242" class="u-url">Enabling the Verification of Computational Results: An Empirical Evaluation of Computational Reproducibility</a></h1>
        <div class="metadata">
            <p class="dateline"><time class="published dt-published" datetime="2018-06-16T00:00:00-04:00" title="16-06-2018">16-06-2018</time></p>  
						<p><a href="mailto:?subject=I%20saw%20this%20and%20thought%20of%20you!&amp;body=Enabling%20the%20Verification%20of%20Computational%20Results:%20An%20Empirical%20Evaluation%20of%20Computational%20Reproducibility;%20https://dl.acm.org/citation.cfm?doid=3214239.3214242" onclick="ga('send', 'social', 'email', 'share', https://dl.acm.org/citation.cfm?doid=3214239.3214242); return true;">Email this article</a></p>
						<p><a href="https://twitter.com/share?url=/directory/directory/Enabling-the-Verification-of-Computational-Results-An-Empiri/);text=Enabling%20the%20Verification%20of%20Computational%20Results:%20An%20Empirical%20Evaluation%20of%20Computational%20Reproducibility,%20https://dl.acm.org/citation.cfm?doid=3214239.3214242" onclick="ga('send', 'social', 'Twitter', 'tweet', /directory/directory/Enabling-the-Verification-of-Computational-Results-An-Empiri/); return true;" target="_blank">Tweet this article</a></p> 
        </div>
    </header><div class="e-content entry-content">
    <p>The ability to independently regenerate published computational claims is widely recognized as a key component of scientific reproducibility. In this article we take a narrow interpretation of this goal, and attempt to regenerate published claims from author-supplied information, including data, code, inputs, and other provided specifications, on a different computational system than that used by the original authors. We are motivated by Claerbout and Donoho's exhortation of the importance of providing complete information for reproducibility of the published claim. We chose the Elsevier journal, the Journal of Computational Physics, which has stated author guidelines that encourage the availability of computational digital artifacts that support scholarly findings. In an IRB approved study at the University of Illinois at Urbana-Champaign (IRB #17329) we gathered artifacts from a sample of authors who published in this journal in 2016 and 2017. We then used the ICERM criteria generated at the 2012 ICERM workshop "Reproducibility in Computational and Experimental Mathematics" to evaluate the sufficiency of the information provided in the publications and the ease with which the digital artifacts afforded computational reproducibility. We find that, for the articles for which we obtained computational artifacts, we could not easily regenerate the findings for 67% of them, and we were unable to easily regenerate all the findings for any of the articles. We then evaluated the artifacts we did obtain (55 of 306 articles) and find that the main barriers to computational reproducibility are inadequate documentation of code, data, and workflow information (70.9%), missing code function and setting information, and missing licensing information (75%). We recommend improvements based on these findings, including the deposit of supporting digital artifacts for reproducibility as a condition of publication, and verification of computational findings via re-execution of the code when possible.</p>
		<div id="index-tags">
		  <ul class="tags">
			Tagged:
		  	<li class="tag"><a href="../categories/reproducible-paper/">reproducible paper</a></li>          
		  </ul>
</div>
    </div>
    </article><article class="h-entry post-text" itemscope="itemscope" itemtype="http://schema.org/Article"><header><h1 class="p-name entry-title"><a href="https://dl.acm.org/citation.cfm?doid=3214239.3214241" class="u-url">Improving Reproducibility of Distributed Computational Experiments</a></h1>
        <div class="metadata">
            <p class="dateline"><time class="published dt-published" datetime="2018-06-16T00:00:00-04:00" title="16-06-2018">16-06-2018</time></p>  
						<p><a href="mailto:?subject=I%20saw%20this%20and%20thought%20of%20you!&amp;body=Improving%20Reproducibility%20of%20Distributed%20Computational%20Experiments;%20https://dl.acm.org/citation.cfm?doid=3214239.3214241" onclick="ga('send', 'social', 'email', 'share', https://dl.acm.org/citation.cfm?doid=3214239.3214241); return true;">Email this article</a></p>
						<p><a href="https://twitter.com/share?url=/directory/directory/Improving-Reproducibility-of-Distributed-Computational-Exper/);text=Improving%20Reproducibility%20of%20Distributed%20Computational%20Experiments,%20https://dl.acm.org/citation.cfm?doid=3214239.3214241" onclick="ga('send', 'social', 'Twitter', 'tweet', /directory/directory/Improving-Reproducibility-of-Distributed-Computational-Exper/); return true;" target="_blank">Tweet this article</a></p> 
        </div>
    </header><div class="e-content entry-content">
    <p>Conference and journal publications increasingly require experiments associated with a submitted article to be repeatable. Authors comply to this requirement by sharing all associated digital artifacts, i.e., code, data, and environment configuration scripts. To ease aggregation of the digital artifacts, several tools have recently emerged that automate the aggregation of digital artifacts by auditing an experiment execution and building a portable container of code, data, and environment. However, current tools only package non-distributed computational experiments. Distributed computational experiments must either be packaged manually or supplemented with sufficient documentation. In this paper, we outline the reproducibility requirements of distributed experiments using a distributed computational science experiment involving use of message-passing interface (MPI), and propose a general method for auditing and repeating distributed experiments. Using Sciunit we show how this method can be implemented. We validate our method with initial experiments showing application re-execution runtime can be improved by 63% with a trade-off of longer run-time on initial audit execution.</p>
		<div id="index-tags">
		  <ul class="tags">
			Tagged:
		  	<li class="tag"><a href="../categories/reproducible-paper/">reproducible paper</a></li>          
		  </ul>
</div>
    </div>
    </article><nav aria-label="Page navigation"><ul class="pagination pagination-lg justify-content-center">
<li class="page-item"><a href="index-15.html" class="page-link" aria-label="Newer posts"><span aria-hidden="true">«</span></a></li>
      <li class="page-item "><a href="." class="page-link">1</a></li>
      <li class="page-item disabled"><a href="#" class="page-link" aria-label="…"><span aria-hidden="true">…</span></a></li>
      <li class="page-item "><a href="index-11.html" class="page-link">12</a></li>
      <li class="page-item "><a href="index-12.html" class="page-link">13</a></li>
      <li class="page-item "><a href="index-13.html" class="page-link">14</a></li>
      <li class="page-item "><a href="index-14.html" class="page-link">15</a></li>
      <li class="page-item "><a href="index-15.html" class="page-link">16</a></li>
      <li class="page-item active"><a href="#" class="page-link">17 <span class="sr-only">(current)</span></a></li>
      <li class="page-item "><a href="index-17.html" class="page-link">18</a></li>
      <li class="page-item "><a href="index-18.html" class="page-link">19</a></li>
      <li class="page-item "><a href="index-19.html" class="page-link">20</a></li>
      <li class="page-item "><a href="index-20.html" class="page-link">21</a></li>
      <li class="page-item "><a href="index-21.html" class="page-link">22</a></li>
      <li class="page-item disabled"><a href="#" class="page-link" aria-label="…"><span aria-hidden="true">…</span></a></li>
      <li class="page-item "><a href="index-78.html" class="page-link">79</a></li>
      <li class="page-item"><a href="index-17.html" class="page-link" aria-label="Older posts"><span aria-hidden="true">»</span></a></li>
  </ul></nav><!--End of body content--><footer id="footer" class="mt-4 mb-4">
            2019 <a href="mailto:vicky.steeves@nyu.edu">NYU Reproducibility WG</a> | <a href="https://github.com/vida-nyu/reproducible-science"><i class="fab fa-github"></i></a> <a href="https://twitter.com/ReproFeed"><i class="fab fa-twitter"></i></a> <a href="https://vida-nyu.github.io/reproducibility-news/feed.rss"><i class="fas fa-rss"></i></a>
            
        </footer>
</div>
</div>


        <script src="../assets/js/all-nocdn.js"></script><!-- fancy dates --><script>
    moment.locale("en");
    fancydates(0, "DD:MM:YYYY");
    </script><!-- end fancy dates --><script>
    baguetteBox.run('div#content', {
        ignoreClass: 'islink',
        captions: function(element) {
            return element.getElementsByTagName('img')[0].alt;
    }});
    </script><script>(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
  ga('create', 'UA-86255717-1', 'auto');
ga('send', 'pageview');</script>
</body>
</html>
